docs/en/oas3/OAS3-PDND-AS.yaml

```
@@ -0,0 +1,313 @@
+            ON JWT.IO</a>
+          required: true
+          schema:
+            type: string
+            format: Signed JWT
+            example: >-
+              DPoP
+              eyJhbGciOiJFUzI1NiIsImtpZCI6ImI4MzlmNGM3LTFlNWQtNGE4YS05ZmM2LTcyZDNiN2YwOTFlYyIsInR5cCI6ImF0K2p3dCJ9.eyJpc3MiOiJpbnRlcm9wLnBhZ29wYS5pdCIsInN1YiI6IjgyOTE0YjNmLTYwYjItNDUyOS1iNGQ2LTNkNGU2N2YwYTkzMyIsImF1ZCI6Imh0dHBzOi8vZXJvZ2F0b3JlLmV4YW1wbGUvZW50ZS1leGFtcGxlL3YxIiwiZXhwIjoxNzMzMDQyMTUwLCJuYmYiOjE3MzMwNDE5NDUsImlhdCI6MTczMzA0MTkyMCwianRpIjoiYzRmNWQ3ZTItYjdjOC00MGY2LTliNmEtZGM5YTRmNWFlYjU3IiwiY2xpZW50X2lkIjoiODI5MTRiM2YtNjBiMi00NTI5LWI0ZDYtM2Q0ZTY3ZjBhOTMzIiwicHVycG9zZUlkIjoiZDJiOWE2NTMtYzQ5Ny00NWM2LWI4ZjEtNWJkZjEyNGM5ZDNhIiwiZGlnZXN0Ijp7ImFsZyI6IlNIQTI1NiIsInZhbHVlIjoiOTkwOGQ5NGI4ZmViMjY4YzAzNzEwNmQ3Yzg5ZTcwNjBjMmNjMWY2YjJiNGViY2I4MDViZmVlNTNhNTM5MzA3YiJ9fQ.9e3dwxSa5fSTPolA2NG7kpa0IEpNY0rM9Wc6cczsKwRlv1fEpKKtjcxnUyRG8fqo5Ajbun_39AqMUqOqCIcCVg
+        - name: DPoP
+          in: header
+          description: >-
+            DPoP proof JWT, to comply with the REST_JWS_2021_POP security
+            pattern. See also <a target="blank"
+            href="https://datatracker.ietf.org/doc/html/rfc9449.html">RFC
+            9449</a>.<br/><br/>
+
+            <a target="blank" href="https://jwt.io/#debugger-io?token=eyJ0eXAiOiJkcG9wK2p3dCIsImFsZyI6IkVTMjU2IiwiandrIjp7Imt0eSI6IkVDIiwia2V5X29wcyI6WyJzaWduIl0sImtpZCI6ImRGVTNNRDI4REpfamZzZmloUHZpMm8tQ3RqTEVVejNwT0lWMEJkTk1mZjgiLCJjcnYiOiJQLTI1NiIsIngiOiJodXlYSVFOdjkwMm9Mc3BYNF96b25DOTRHNnlFbG42bHNkbS0xd003MzJvIiwieSI6Ikk5UERFYXdXSHFhRkRHeDFaa05rLTJQVjZXZHBjYUgzQWZPYkJTTGloZ3cifX0.eyJqdGkiOiItQndDM0VTYzZhY2MybFRjIiwiaHRtIjoiUE9TVCIsImF0aCI6ImM1MmMzZTU4MmNjYTRhZTgyYTRhYWRjODA3MTIwZGZmODRlNmIzYjA5MTg3NjMwMmYxY2QyN2I0ODRmNmE1YjgiLCJodHUiOiJodHRwczovL3NlcnZlci5leGFtcGxlLmNvbS90b2tlbiIsImlhdCI6MTc2MjI2MjYxNn0.C--7xxs9hbF6eA6HM9gGvQ1FtlseiJ9hYFAsMnAsu-JtlzNXJXoAvrVOVkeYtkQ6beXveOern56_rLDoR4gmiw">EXAMPLE
+            ON JWT.IO</a>
+          required: true
+          schema:
+            type: string
+            format: JWT
+            example: >-
+              eyJ0eXAiOiJkcG9wK2p3dCIsImFsZyI6IkVTMjU2IiwiandrIjp7Imt0eSI6IkVDIiwia2V5X29wcyI6WyJzaWduIl0sImtpZCI6ImRGVTNNRDI4REpfamZzZmloUHZpMm8tQ3RqTEVVejNwT0lWMEJkTk1mZjgiLCJjcnYiOiJQLTI1NiIsIngiOiJodXlYSVFOdjkwMm9Mc3BYNF96b25DOTRHNnlFbG42bHNkbS0xd003MzJvIiwieSI6Ikk5UERFYXdXSHFhRkRHeDFaa05rLTJQVjZXZHBjYUgzQWZPYkJTTGloZ3cifX0.eyJqdGkiOiItQndDM0VTYzZhY2MybFRjIiwiaHRtIjoiUE9TVCIsImF0aCI6ImEyMWQxMzUzZGYwZTExOTQ4ODE1NDAwZGU2MjUxYzUwNjlhYTFhNjBjMDg5YWViYTQ2MTI1Y2JlNjhmNzQ1ZmYiLCJodHUiOiJodHRwczovL3NlcnZlci5leGFtcGxlLmNvbS90b2tlbiIsImlhdCI6MTc2MjI2MjYxNn0.bVf2E4qvHTNGAGXPNcOLrgI_QcXmx0k8Ays6d6gPuT6X1Pbwzz_ltYBxiT28O8ab2-ulk9Rg8TIHOG_H5ZCCSA
```

See previous comment.

```suggestion
              eyJ0eXAiOiJkcG9wK2p3dCIsImFsZyI6IkVTMjU2IiwiandrIjp7Imt0eSI6IkVDIiwia2V5X29wcyI6WyJzaWduIl0sImtpZCI6IjM5ZmE5NjBiLTc3M2YtNDllZi04YTBlLWU3NzNlOWI5N2FlOCIsImNydiI6IlAtMjU2IiwieCI6Imh1eVhJUU52OTAyb0xzcFg0X3pvbkM5NEc2eUVsbjZsc2RtLTF3TTczMm8iLCJ5IjoiSTlQREVhd1dIcWFGREd4MVprTmstMlBWNldkcGNhSDNBZk9iQlNMaWhndyJ9fQ.eyJqdGkiOiIyYzc2ZmNhMy1jYjRlLTQzMTItOGI2ZS05NzQ5NDYyZjQyMGQiLCJodG0iOiJQT1NUIiwiYXRoIjoiQkd6SUxmUVBnaTNEVjZPUjNSNGZSOFVyREUzUEJ5c1RsUkphSGhLTkx1dyIsImh0dSI6Imh0dHBzOi8vYXV0aGVudGljLXNvdXJjZS5leGFtcGxlLml0IiwiaWF0IjoxNzYyMjYyNjE2fQ.gCZIfiuu8knBNNAqPsVuZKdIsvGUauNb0ZO_FxqfWDRbvIsM_eimfNgPfkXzHfPeyluxJjpvf5e5q7v27gmGJQ
```